metrics-proxy/src/main/java/ai/vespa/metricsproxy/metric/model/json/GenericJsonModel.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+ */
+
+package ai.vespa.metricsproxy.metric.model.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_ABSENT;
+
+/**
+ * @author gjoranv
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(NON_ABSENT)
+@JsonPropertyOrder({ "node", "services" })
+public class GenericJsonModel {
+    private static Logger log = Logger.getLogger(GenericJsonModel.class.getName());
+
+    @JsonProperty("node")
+    public GenericNode node;
+
+    @JsonProperty("services")
+    public List<GenericService> services;
+
+    public String serialize() {
+        ObjectMapper mapper = JacksonUtil.createObjectMapper();
+        try {
+            return mapper.writeValueAsString(this);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Fyi, the log() overloads taking a throwable will also print the stack trace to the log.